glide.lock

```diff
@@ -895,7 +918,7 @@ imports:
   subpackages:
   - imageprogress
 - name: github.com/openshift/source-to-image
-  version: e3e0eedc3f9dd55377ca2b8444769ec2ea54c895
+  version: 5bf2cb171c2cab1270fba80dc53ef4ff25b12c97
```

Comment:
I wouldn't do that. This might cause some weird problems. If possible pin s2i to the current version we have. It can be easily bumped after rebase lands. Unless you strictly required something from newer version.

Reply:
> I wouldn't do that. This might cause some weird problems. If possible pin s2i to the current version we have. It can be easily bumped after rebase lands. Unless you strictly required something from newer version.

seems to be working.